py/autoevals/moderation.py

```diff
@@ -0,0 +1,59 @@
+from braintrust_core.score import Score, Scorer
+
+from .oai import run_cached_request
+
+REQUEST_TYPE = "moderation"
+
+
+class Moderation(Scorer):
```

Comment:
I just landed https://github.com/braintrustdata/autoevals/pull/60

which introduces an `OpenAIScorer` that you can inherit from, and delegate dealing with the auth params to.

Reply:
Nice

Reply:
Updated: https://github.com/braintrustdata/autoevals/pull/63/commits/10a36d9785a7428d7c8db3a9385367d10282dae6#diff-059564dc34fe7366598e59c4b9fbcab770ba4689f9c9418010178055baed560eR10